app/graphql/graphql_devise/mutations/resend_confirmation.rb

```diff
@@ -0,0 +1,30 @@
+module GraphqlDevise
+  module Mutations
+    class ResendConfirmation < Base
+      argument :email,        String, required: true, prepare: ->(email, _) { email.downcase }
+      argument :redirect_url, String, required: true
+      
+      field :success, Boolean, null: false
+      field :message, String, null: false
+      
+      def resolve(email:, redirect_url:)
+        resource = controller.find_resource(:uid, email)
+
+        if resource
+          yield resource if block_given?
+          resource.send_confirmation_instructions({
+            redirect_url: redirect_url,
+            template_path: ['graphql_devise/mailer']
+          })
+
+          {
+            success: true,
+            message: I18n.t('graphql_devise.confirmations.send_instructions', email: email)
+          }
```

Comment:
you are forgetting the `authenticable` key on the hash which I think is added as a field to this mutation too.

`{ authenticable: resource }`

Reply:
I can add that. I wasn't sure if it was needed since you're just resending a confirmation to the email you passed. I'll add it.